code/modules/surgery/organs/parasites.dm

```diff
@@ -0,0 +1,35 @@
+/obj/item/organ/internal/body_egg/spider_eggs
+	name = "spider eggs"
+	icon = 'icons/effects/effects.dmi'
+	icon_state = "eggs"
+	var/stage = 1
+
+/obj/item/organ/internal/body_egg/spider_eggs/on_life()
+	if(stage < 5 && prob(3))
```

Comment:
I'd agree with @Tastyfish here. One of the benefits of inheritance is that if you know where things are done in one type, you don't need to hunt around for it in other types of the same class. It just makes for cleaner code.

Just use an `if(owner.stat == "DEAD") return` at the start of the `egg_process()`